### PR TITLE
add notation module

### LINKED ISSUE #14: add notation module
1. create rank entity:

- Conference.id
- User.id
- note (max 5)

2. create column average ranking in conference/all & conference/show/{id}

3. possibility for a user to rate a conference ( /5)

### PR DESCRIPTION
**Pull request by Ryan-Labs**
Fixes #14 

without condition 5 max on value attribute

### CODE REVIEW

File: templates/conference/index.html.twig [hunk 1]
```diff
@@ -13,6 +13,7 @@
                 <th>Description</th>
                 <th>Date</th>
                 <th>actions</th>
+                <th>note</th>
```
Comment (AitZak) [inline templates/conference/index.html.twig]: Add Uppercase for the first letter of  `actions` and `note`.

File: templates/conference/index.html.twig [hunk 2]
```diff
@@ -13,6 +13,7 @@
                 <th>Description</th>
                 <th>Date</th>
                 <th>actions</th>
```
Comment (AitZak) [inline templates/conference/index.html.twig]: I suggest to display note before actions

File: src/Entity/Rating.php [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    /**
+     * @ORM\Column(type="float")
+     */
+    private $value;
+
+    public function getId(): ?int
+    {
+        return $this->id;
+    }
+
```
Comment (AitZak) [inline src/Entity/Rating.php]: Please delete `getId()` since there is no `id` in your entity

File: templates/conference/index.html.twig (hunk 2 above)
Comment (Ryan-Labs) [reply templates/conference/index.html.twig]: good suggestion

**Pull request merged**